CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 ###### Dev
 
+-   Converts relative to absolute paths for imports - luc
```

Comment:
We shipped this beta already - add your changelog notes inside  the master section for the next release 👍